packages/nextjs/app/builders/0xb36faaA498D6E40Ee030fF651330aefD1b8D24D2/page.tsx

```diff
@@ -0,0 +1,214 @@
+            <div className="relative w-40 h-40">
+              <div className="absolute inset-0 bg-gradient-to-r from-primary via-secondary to-accent rounded-full animate-pulse" />
+              <div className="absolute inset-[3px] bg-base-100 rounded-full" />
+              <div className="relative w-full h-full rounded-full overflow-hidden">
+                <Image
+                  src="/builders/lordheb.eth.jpg"
+                  alt="Ihab Heb - Blockchain & AI Developer"
+                  fill
+                  sizes="160px"
+                  className="object-cover"
+                  priority
+                />
+              </div>
+            </div>
+            <div className="text-center space-y-3">
+              <h1 className="text-5xl font-bold bg-clip-text text-transparent bg-gradient-to-r from-primary to-secondary drop-shadow-lg">
+                Ihab Heb
+              </h1>
+              <p className="text-xl text-base-content/80 font-mono">lordheb.eth</p>
+              <p className="text-lg text-base-content/60">Blockchain & AI Developer</p>
+            </div>
+
+            {/* Social Links  */}
+            <div className="flex justify-center gap-8 pt-4">
+              {socials.map(({ icon: Icon, url }) => (
+                <Link
+                  key={url}
+                  href={url}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  className="text-base-content/70 hover:text-primary transition-all duration-300 transform hover:scale-110 hover:-translate-y-1"
+                >
+                  <Icon className="w-7 h-7" />
+                </Link>
+              ))}
+            </div>
+          </div>
+        </div>
+
+        {/* Bio Section */}
+        <div className="bg-base-100 rounded-2xl p-8 shadow-lg space-y-6 border border-base-300">
+          <h2 className="text-2xl font-bold text-base-content flex items-center gap-2">
+            <SiBlockchaindotcom className="text-primary" />
+            About Me
+          </h2>
+          <p className="text-lg text-base-content/80 leading-relaxed">
+            Blockchain & AI Developer contributing to the effective acceleration & interoperability of the web3 space,
+            digital public goods and manifesting the futuristic vision of the peoples internet.
+          </p>
+
+          <div className="space-y-3">
+            <h3 className="text-xl font-semibold text-base-content">Tech Stack</h3>
+            <div className="flex flex-wrap gap-3">
+              {["Solidity", "TypeScript", "Next.js", "Foundry", "AI Agents Framework", "Rust", "Cairo"].map(tech => (
+                <span
+                  key={tech}
+                  className="px-4 py-2 text-base bg-base-200 rounded-lg text-base-content/70 hover:text-primary hover:bg-base-300 transition-all duration-300 hover:scale-105"
+                >
+                  {tech}
+                </span>
```

Comment:
There are two problems related to the hover effect:  
- In dark mode, the text becomes invisible.  
- Do we really want a hover effect here anyway? Usually, hover effects are for interactive elements, which is not the case here.